sdk/batch/Microsoft.Azure.Batch/Tools/CodeGenerationLibrary/AccessModifier.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CodeGenerationLibrary
+{
+    public enum AccessModifier
+    {
+        @public = 4,
```

Comment:
Aren't enum values usually UpperCamelCase by convention? The `@` looks a bit odd here.

Also seems like it might be better to use a string enum here since we're serializing to/from JSON strings. I realize it's a bit nice to be able to say things like `access > AccessModifier.@protected`, but IMO that's a pretty minor benefit (and it's not completely clear that all access modifiers can be strictly ordered like that anyway)

Reply:
Also, if we're trying to order them, internal is more permissive than private

Reply:
> Aren't enum values usually UpperCamelCase by convention? The @ looks a bit odd here.

Good call. I was matching the enum's value so that the output could just ToString() it (the @s are required since public, private, etc., are protected keywords), but I didn't think about the fact that broke convention.